addons/l10n_rw/data/template/account.fiscal.position-rw.csv

```diff
@@ -0,0 +1,7 @@
+"id","name","sequence","auto_apply","vat_required","country_id","country_group_id","tax_ids/tax_src_id","tax_ids/tax_dest_id"
+"fiscal_position_rw_exempt","Exempt taxpayer","1","","","","","VAT_S_IN_RW_18","VAT_S_exempt_O"
+"","","","","","","","VAT_S_IN_O","VAT_S_exempt_O"
+"","","","","","","","VAT_P_IN_RW_18","VAT_P_IN_RW_0"
```

Comment:
I'm not sure I understand this case, why would we map from 18% to 0% when the seller is exempted, does it not depend on the buyer?

Reply:
We decided with laura to remove the fiscal position :smile: Thanks !